app/config/collections.php

```diff
@@ -2668,15 +2690,15 @@
                 'filters' => [],
             ],
             [
-                '$id' => 'openSSLVersion',
+                '$id' => 'fileSecret',
```

Comment:
Same, we should store a list of keys not a single one.

Reply:
we are just using keys per file, so I don't think we need arrays here? Because if/when we change the key the file should already be encrypted using the new key.